src/settings/DisplaySettings.ts

```diff
@@ -0,0 +1,6 @@
+export class DisplaySettings {
+    public static readonly CANVAS_MIN_MARGIN_PX: number = 20;
```

Comment:
to jest ten margines który rozrysowałem na kartce? (u mnie nazywał się paddingiem)
jeśli tak, to ja bym założył procentową wartość a nie pikselową...
generalnie założenie jest takie: co się da zrobić skalowalne (względem zoomu) to to róbmy. PX są sztywne i trochę upierdliwe mogą się później zrobić :/

Reply:
Może do tego jeszcze wrócimy, ale hm... nom ja bym to chciał mieć stałe. Trzeba pogadać.